### PR TITLE
use bugfixed mocha version

### PR DESCRIPTION
**Pull request by cdietrich**
use bugfixed mocha version
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.web/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"name": "xtext",
 	"private": true,
 	"devDependencies": {
-		"mocha": "6.1.4",
+		"mocha": "^7.2.0",
```
Comment (miklossy) [inline org.eclipse.xtext.web/package.json:5]: Why is the `^` symbol needed?

Comment (cdietrich) [reply org.eclipse.xtext.web/package.json:5]: ask npm

Comment (miklossy) [reply org.eclipse.xtext.web/package.json:5]: I do not know much about npm, but I rely on you.

Comment (miklossy) [reply org.eclipse.xtext.web/package.json:5]: Are the changes in the `package-lock.json` file necessary/on purpose?

**Comment by cdietrich** [comment]
file is generated by npm

**Comment by miklossy** [comment]
Than it is fine for me.

**Pull request merged**